src/br/com/zup/nossacasacodigo/author/Author.java

```diff
@@ -1,26 +1,28 @@
 package br.com.zup.nossacasacodigo.author;
 
-import java.util.Date;
-
-import br.com.zup.nossacasacodigo.controllers.AuthorService;
-import br.com.zup.nossacasacodigo.controllers.AuthorsInformarionValidators;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class Author {
 	private String name;
 	private String email;
 	private String description;
-	private Date createdAt;
+	private LocalDateTime createdAt;
 	
-	public Author(String name, String email, String description, Date createdAt) {		
-		AuthorsInformarionValidators.emailValidator(email);
-		AuthorsInformarionValidators.nameValidator(name);
-		AuthorsInformarionValidators.descriptionValidator(description);
-		AuthorsInformarionValidators.dateValidator(createdAt);
+	public Author(String name, String email, String description) {		
+		emailValidator(email);
+		nameValidator(name);
+		descriptionValidator(description);
+		
+		LocalDateTime dataTime = getDateTime();
+		dateValidator(dataTime);
 		
 		this.name = name;
 		this.email = email;
 		this.description = description;
-		this.createdAt = createdAt;
+		this.createdAt = dataTime;
```

Comment:
Nesse ponto, você pode fazer algo como `this.createdAt = LocalDateTime.now()`. Como a `LocalDateTime` é uma classe do Java, já temos a garantia que ela funciona do jeito esperado :D